docs_user/modules/proc_adopting-the-networking-service.adoc

```diff
@@ -2,11 +2,14 @@
 
 = Adopting the {networking_service}
 
-Adopting {networking_first_ref} means that an existing `OpenStackControlPlane` custom resource (CR), where the {networking_service}
-is supposed to be disabled, should be patched to start the service with the
-configuration parameters provided by the source environment.
+To adopt the {networking_first_ref}, you patch an existing `OpenStackControlPlane` custom resource (CR), where the {networking_service}
+is disabled. The patch starts the service with the
+configuration parameters that are provided by the {rhos_prev_long} ({OpenStackShort}) environment.
 
-When the procedure is over, the expectation is to see the `NeutronAPI` service is running: the {identity_service_first_ref} endpoints should be updated and the same backend of the source Cloud will be available. If the conditions above are met, the adoption is considered concluded.
+The {networking_service} adoption is complete if you see the following results:
+
+* The `NeutronAPI` service is running.
+* The {identity_service_first_ref} endpoints are updated, and the same back end of the source cloud is available.
 
 This guide also assumes that:
```

Comment:
@slawqo Can you help me understand how this part is relevant to the procedure?

Also, what is meant by “one side” and “other side”?  Do you mean "source environment (RHOSP)" and "target environment (the OpenShift cluster)"?

Reply:
I'm not sure what exactly You are asking for. This says about how to start adoption of the networking service (patch existing CR where neutron service was disabled to enable it) and how to validate that this adoption was finished (check that NeutronAPI is running and networking endpoints in the keystone db points to the new neutron service in the OpenShift cluster, and not the one in the OSP 17.1)

Where do You see "one side" and "other side" there? I don't see it so it is hard for me to tell what it means.

Reply:
@slawqo I apologize for the confusion. My comment applies to the following text:
"This guide also assumes that:
. A {OpenStackPreviousInstaller} environment (the source Cloud) is running on one side;
. A `SNO` / `CodeReadyContainers` is running on the other side."

Reply:
yes, so the 'one side' is actually old, tripleo/Director based deployment and 'the other side' is in this context is Openshift cluster (it can be CRC in e.g. u/s ci)